pkg/monitor/monitor/util.go

```diff
@@ -139,7 +139,7 @@ func getGrafanaEnvs() []core.EnvVar {
 	}
 }
 
-func getAlertManagerRulesVersion(tc *v1alpha1.TidbCluster, monitor *v1alpha1.TidbMonitor) string {
```

Comment:
I have add unit test for the case of cluster field not specified , i think this piece of code not need to add unit test cover it . because i just optimize input parameter.